docs/audits/total-ad-blocking-time.md

```diff
@@ -0,0 +1,18 @@
+# Reduce Ad JS Blocking Time
+
+## Overview
+
+This audit measures how much [blocking time](https://web.dev/tbt/) is
```

Comment:
Suggest rewording to: "The amount of blocking time attributable to ad-related scripts."